src/app/core/services/issue-comment.service.ts

```diff
@@ -16,11 +18,7 @@ export class IssueCommentService {
   }
 
   getIssueComments(issueId: number): Observable<IssueComments> {
-    if (!this.comments.get(issueId)) {
-      return this.initializeIssueComments(issueId);
-    } else {
-      return of(this.comments.get(issueId));
-    }
+    return this.initializeIssueComments(issueId);
```

Comment:
Can make an issue for this? Because ideally we shouldnt query github everytime github is this functuon is called. We can work on it in the future.